Add two PHE global redirects

marksheldonGDS opened the pull request:
https://govuk.zendesk.com/agent/tickets/1192660

Public Health England have asked us to configure these domains to global archives. The sites have been decommissioned and should point users to the PHE homepage on GOV.UK

* www.foodwm.org.uk 
* www.obesitywm.org.uk

I've used the latest TNA timestamps provided on the Zendesk ticket.

jennyd commented:
These are configured as global archives, but the commit messages say they're global redirects - could you update the commit messages?

jennyd commented:
These sites look identical in the TNA crawls. The timestamps are less than an hour apart, and they work interchangeably on all four domains, so we could simplify this by configuring them all as a single site, using one of the timestamps.

marksheldonGDS commented:
Squashed these into a single commit and changed the message to reflect what we're actually doing. Branch name and PR title both still mention 'redirect' but these are configured as archives.
